tests/test_reverse_m2m_relation_crudl_operations_are_supported.py

```
@@ -0,0 +1,242 @@
+    )
+    book = models.Book.objects.create(
+        title="Some book",
+        isbn="9783161484100",
+        publication_date="2021-01-01",
+        publisher=publisher,
+    )
+    book.authors.set([author_1, author_2])
+
+    response = client.get("/api/authors")
+    assert response.status_code == status.HTTP_200_OK, response.json()
+    assert len(response.json()) == 2
```

Ruff (0.8.2)

Magic value used in comparison, consider replacing `2` with a constant variable

(PLR2004)